setup.cfg

```diff
@@ -35,7 +35,8 @@ project_urls =
 [options]
 packages =
     SPARQLWrapper
-include_package_data = True
+include_package_data =
+    SPARQLWrapper = py.typed
```

Comment:
[`include_package_data` is a boolean](https://setuptools.pypa.io/en/latest/userguide/datafiles.html), if you want to include specific files you should use [`package_data`](https://setuptools.pypa.io/en/latest/userguide/datafiles.html)
```suggestion
```

Reply:
see suggestion regarding `package_data`